abis/README.md

```diff
@@ -1,4 +0,0 @@
-# ABI
-ABI stands for application binary interface, is basically how you call functions in a contract and get data back.
```

Comment:
shouldn't we keep this in bridge/abi?

Reply:
not necesary, and we are deleting the abi folder and re creating it on compile so it will get deleted

Reply:
I meant the explanation of what an ABI is, which I think we're losing

Reply:
yeah im loosing on pupose, at this point anyone who needs to use it can get it from the internet xD

Reply:
hahaha fair enough